sdlib/d/gc/region.d

```diff
@@ -63,6 +65,11 @@ private:
 	size_t minAddress = AddressSpace;
 	size_t maxAddress = 0;
 
+	// Count of blocks that are currently allocated.
```

Comment:
allocated here is not consistent with the acquire/release vocabulary that is used. These block might be mostly empty, as far as we know.